src/skeletons.tsx

```diff
@@ -0,0 +1,184 @@
+    width = '100%',
+    disableAnimation = false,
+    ariaLabel,
+}: SkeletonLineProps): JSX.Element => {
+    const animationClasses = useAnimation({disableAnimation});
+    const styleClass = useSkeletonStyles({width});
+
+    return (
+        <div
+            className={`${animationClasses.animation} ${styleClass.wrap}`}
+            role="status"
+            aria-hidden={ariaLabel === undefined}
+            aria-busy={ariaLabel !== undefined}
+            aria-label={ariaLabel}
+        >
+            <SkeletonBase></SkeletonBase>
+        </div>
+    );
+};
+
+type SkeletonTextProps = Omit<SkeletonLineProps, 'width'>;
+
+const SkeletonText = ({disableAnimation = false, ariaLabel}: SkeletonTextProps): JSX.Element => {
+    const animationClasses = useAnimation({disableAnimation});
+    const styleClass = useSkeletonStyles({height: 'fit-content'});
+
+    return (
+        <Stack
+            className={`${animationClasses.animation} ${styleClass.wrap}`}
+            space={16}
+            role="status"
+            aria-hidden={ariaLabel === undefined}
+            aria-busy={ariaLabel !== undefined}
+            aria-label={ariaLabel}
+        >
+            <SkeletonBase />
+            <SkeletonBase />
+            <SkeletonBase width="75%" />
+        </Stack>
+    );
+};
+
+type SkeletonCircleProps = Omit<SkeletonLineProps, 'width'> & {size?: number | string};
+
+const SkeletonCircle = ({
+    disableAnimation = false,
+    ariaLabel,
+    size = 40,
+}: SkeletonCircleProps): JSX.Element => {
+    const animationClasses = useAnimation({disableAnimation});
+    const styleClass = useSkeletonStyles({width: size, height: size});
+
+    return (
+        <div
+            className={classNames(animationClasses.animation, styleClass.wrap)}
+            role="status"
+            aria-hidden={ariaLabel === undefined}
+            aria-busy={ariaLabel !== undefined}
+            aria-label={ariaLabel}
+        >
+            <SkeletonBase height={size} width={size} radius="50%" />
+        </div>
+    );
+};
+
+const useSkeletonRowStyles = createUseStyles(() => ({
+    row: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    circle: {
+        flexShrink: 0,
+        flexGrow: 0,
+    },
+    line: {
+        flexShrink: 1,
+        marginLeft: 16,
+    },
+}));
+
+type SkeletonRowProps = SkeletonLineProps;
+
+const SkeletonRow = ({
+    width = '100%',
+    disableAnimation = false,
+    ariaLabel,
+}: SkeletonRowProps): JSX.Element => {
+    const animationClasses = useAnimation({disableAnimation});
+    const styleClass = useSkeletonStyles({width, height: 'content-initial'});
+    const rowClass = useSkeletonRowStyles();
+
+    return (
+        <div
+            className={classNames(animationClasses.animation, styleClass.wrap, rowClass.row)}
+            role="status"
+            aria-hidden={ariaLabel === undefined}
+            aria-busy={ariaLabel !== undefined}
```

Comment:
shouldn't `aria-busy` be always `true` for skeletons?

Reply:
not really, at least in my mind: if it's hidden, then the component should be totally ignored by the assistant. But if we pass a label, it means we want to give it a semantic meaning, and then we do want to be showed!